shopfloor/services/cluster_picking.py

```diff
@@ -406,9 +411,10 @@ def _data_move_line(self, line, **kw):
         data["batch"] = self.data.picking_batch(batch)
         data["picking"] = self.data.picking(picking)
         data["postponed"] = line.shopfloor_postponed
-        data["product"]["qty_available"] = product.with_context(
-            location=line.location_id.id
-        ).qty_available
+        if "no_qty_available" not in kw or not kw.get("no_qty_available"):
```

Comment:
why not simply this?
```suggestion
        if not kw.get("no_qty_available"):
```

Reply:
@sbejaoui